rendercv/cli/utilities.py

```diff
@@ -260,3 +260,35 @@ def parse_render_command_override_arguments(
         key_and_values[key] = value
 
     return key_and_values
+
+
+def build_rendercv_settings(
```

Comment:
We may need to udpate this according to https://github.com/sinaatalay/rendercv/pull/166#discussion_r1740264333.

Also, I would suggest naming this: `update_render_settings`.

And we don't need to store default values in another dict. See `inspect` module of Python. It can read functions and return the default values of the arguments. If we don't use this, every time we change a default value, we will be in trouble.

Reply:
Should I change the variable name of the function to keep consistency with the data model?

If that is okay. then I'll do this:

- `dont_generate_markdown` variable will be called `no_markdown`
- `dont_generate_html` variable will be called `no_html`
- `dont_generate_png` variable will be called `no_png`

These changes will be present in the code only. or should I change the rendercv settings blocks definition?

Reply:
I prefer the old names. We can update the YAML file inputs with the old names.